devsetup/tripleo/generate_ceph_inventory.py

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env python3
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+import argparse
+import re
+import sys
+import yaml
+
+
+def parse_opts(argv):
+    parser = argparse.ArgumentParser(
+            description='Create an ansible inventory from a yaml file similar'
+                        'to the one output by `openstack overcloud node provision`.'
```

Comment:
We need this script because "deployed ceph" works with the output of `openstack overcloud node provision` but we do not have metalsmith here. Thus, we're using the script to give us what metalsmith would have given us.